### PR TITLE
Add gs:// path loading to weights

### PR DESCRIPTION
**Pull request by LukeWood**
Caveats:
- no file hash
- potential for bad caching

### CODE REVIEW

File: keras_cv/models/weights.py [hunk 1]
```diff
@@ -15,6 +15,12 @@
 
 
 def parse_weights(weights, include_top, model_type):
+    if weights.startswith("gs://"):
+        weights = weights.replace("gs://", "https://storage.googleapis.com/")
+        return data_utils.get_file(
+            origin=weights,
+            cache_subdir="custom_models",
```
Comment (ianstenbit) [inline keras_cv/models/weights.py:22]: probably just "models" to be consistent with the equivalent on line 33

**Review by ianstenbit — APPROVED** [review]
Thanks Luke!

**Comment by jbischof** [comment]
What's the motivation for this change?

**Comment by LukeWood** [comment]
> What's the motivation for this change?

To support:
```
ResNet50(weights='gs://your-custom-path')
```

This makes loading weights in a sidecar evaluator or basically anywhere else that is not on the same host machine a lot easier.

**Pull request merged**